host/frontend/host-orchestrator/instancemanager_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestCreateCVDInvalidRequests(t *testing.T) {
+	im := &InstanceManager{}
+	var validRequest = func() *CreateCVDRequest {
+		return &CreateCVDRequest{
+			BuildInfo: &BuildInfo{
+				BuildID: "1234",
+				Target:  "aosp_cf_x86_64_phone-userdebug",
+			},
+			FetchCVDBuildID:     "9999",
+			BuildAPIAccessToken: "fakeaccesstoken",
+		}
+	}
+	// Make sure the valid request is indeed valid.
+	_, err := im.CreateCVD(validRequest())
```

Comment:
Actually, you should probably only call validateRequest in this test, because if this test ever fails with a real implementation of the IM a device will probably be created, which is not a good side effect of a failed test.

Reply:
good catch! thanks. used only validateRequest.